packages/fxa-auth-server/lib/senders/templates/partials/appBadges.html

```diff
@@ -0,0 +1,18 @@
+<tr style="page-break-before: always">
+  <td align="center" style="padding: 40px 0 10px;">
+    <a href="{{{iosUrl}}}">
+      <img src="https://accounts.firefox.com/images/apple_app_store_button/en.svg" width="152" height="45" alt="" style="-ms-interpolation-mode: bicubic;" />
```

Comment:
Why is `https://accounts.firefox.com/` hardcoded here?

Reply:
These are app store badges, ideally should be in the cdn but I couldn't get those changes in time.